pyEpiabm/pyEpiabm/core/household.py

```diff
@@ -104,3 +106,26 @@ def remove_household(self):
         """
         self.microcell.households.remove(self)
         self.cell.households.remove(self)
+
+    def set_id(self, id: str):
+        """Updates ID of household (i.e. for input from file).
```

Comment:
Style point - you are referring to a single household so use the appropriate article - otherwise you are referring to an entire class (Cell, Microcell and Household), which are always capitalized in the docs